src/main/java/mate/academy/model/User.java

```diff
@@ -34,7 +35,11 @@ public class User implements UserDetails {
     @Column(nullable = false)
     private String lastName;
     private String shippingAddress;
-    @ManyToMany
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @ManyToMany
```
you can solve it by using EntityGraph
Just add 

```
@EntityGraph(attributePaths = "roles")
Optional<User> findByEmail(String email);
```

Reply:
thanks